omod/src/main/java/org/openmrs/module/patientflag_rest/web/wrapper/PatientBean.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * MozView Technologies, Lda. 2010 - 2016
+ */
+package org.openmrs.module.patientflag_rest.web.wrapper;
+
+import java.io.Serializable;
+
+/**
+ * @author Stélio Moiane
+ *
+ */
+public class PatientBean implements Serializable {
```

Comment:
Investigate whether we actually need this bean, since it doesn't seem to exist in other modules that implement RESTful resources.